### PR TITLE
Updated cert URL

### PR DESCRIPTION
**Pull request by cschroed-usgs**
Updated cert URL, and the intentions for root CA secure delivery based on conversations on the SSL Intercept Team's mailing list.

### CODE REVIEW

File: ssl/UsingGitWithHTTPSInterception.md [hunk 1]
```diff
@@ -35,7 +35,7 @@ HTTPS is a secure protocol, but it is not as secure as SSH (see [note 2](#note-2
        * Using wired internet at home without using the VPN
        * Using wifi at a coffee shop without using the VPN
 * Download the new DOI Root Certificate Authority certificate from here
- * http://blockpage.doi.gov/images/DOIRootCA.crt
+ * http://sslhelp.doi.net/docs/DOIRootCA2.cer
```
Comment (cschroed-usgs) [inline ssl/UsingGitWithHTTPSInterception.md]: @emartinez-usgs I remember you were having trouble connecting to this host to retrieve the cert. Do the suspected DNS issues persist?

Comment (jmfee-usgs) [inline ssl/UsingGitWithHTTPSInterception.md]: The blockpage site doesn't work for me, but sslhelp does.

**Comment by cschroed-usgs** [comment]
This addresses #40

File: ssl/UsingGitWithHTTPSInterception.md (hunk 1 above)
Comment (cschroed-usgs) [inline ssl/UsingGitWithHTTPSInterception.md]: Thanks for confirming @jmfee-usgs !

**Comment by cschroed-usgs** [comment]
Thanks @emartinez-usgs !

**Pull request merged**